app/lib/account/default_auth_provider.dart

```diff
@@ -64,16 +64,25 @@ class DefaultAuthProvider extends AuthProvider {
       // without any fallback (e.g. authenticating the token as a User).
 
       // TODO: use the tokeninfo endpoint instead
-      await _verifyToken(idToken, openIdDataFetch: fetchGoogleCloudOpenIdData);
+      await _verifyToken(
+        idToken,
+        openIdDataFetch: fetchGoogleCloudOpenIdData,
+        maySkipNbfCheck: true,
```

Comment:
Do we need to make `nbf` required in the github case?

If signed by github or gcp, and it includes nbf or not includes nbf can't we trust it either way?

It's not like the caller can remove the claim if it was there.